Generate a hash function when `frozen` is `True` (#1881)
samuelcolvin/pydantic: changes/1880-rhuille.md

> Add a new `frozen` boolean parameters. Setting `frozen=True` does everything that `allow_mutation=False` does, and also generates a `__hash__()` method for the model. This makes instances of the model potentially hashable if all the attributes are hashable. (default: `False`)

Reviewer: ```suggestion
Add a new `frozen` boolean parameter to `Config` (default: `False`).
Setting `frozen=True` does everything that `allow_mutation=False` does, and also generates a `__hash__()` method for the model. This makes instances of the model potentially hashable if all the attributes are hashable.
```

Author: https://github.com/samuelcolvin/pydantic/pull/1881/commits/7397b678af3ef098d7e038a185e22f4edd973dfc

> Add a new `frozen` boolean parameters. Setting `frozen=True` does everything that `allow_mutation=False` does, and also generate a hash function for the model i.e. `__hash__` is not `None`. This makes instances of the model potentially hashable if all the attributes are hashable. (default: `False`)

Reviewer: ```suggestion
Add a new `frozen` boolean parameters. Setting `frozen=True` does everything that `allow_mutation=False` does, and also generates a `__hash__()` method for the model. This makes instances of the model potentially hashable if all the attributes are hashable. (default: `False`)
```

Author: https://github.com/samuelcolvin/pydantic/pull/1881/commits/0d6eaec4bebb9e06fc2042f175a9c6fbf30deb18
